filecoin-proofs/src/bin/phase2.rs

```diff
@@ -321,10 +267,9 @@ fn blank_window_post_poseidon_circuit(
 
     let public_params = window_post_public_params(&post_config).unwrap();
 
-    <FallbackPoStCompound<PoseidonHasher> as CompoundProof<
-        Bls12,
-        FallbackPoSt<PoseidonHasher>,
-        FallbackPoStCircuit<Bls12, PoseidonHasher>,
+    <FallbackPoStCompound<DefaultOctTree> as CompoundProof<
+        FallbackPoSt<DefaultOctTree>,
+        FallbackPoStCircuit<DefaultOctTree>,
```

Comment:
Again, maybe I'm missing how this works, but hard-coded tree shapes in a function of sector size worry me.

All of these should be considered BLOCKING — requiring either a change or an explanation.